requirements/cross-chain-transfer.md

```diff
@@ -77,15 +78,20 @@ Assuming members are synced and have the same view of the multisig address (the
 It may be that in order to use a multisig UTXO the algorithm will require it to be buried under enough blocks, to avoid reorgs and ensure that all member have the same view of the UTXO list.
 
 **The algorithm to select UTXOs for a transfer:**
+
 - The oldest UTXO (determined by block height) is selected first.
 - If a block has more than one UTXO, then they are selected in order of appearance in the block transaction list.
 - If multiple UTXOs appear within a transaction then they are selected in ascending index order.
+- Dusty UTXOs are ignored (below 100 Strats ?).
```

Comment:
This figure seems high to me, although it's difficult to say definitively. I suppose there is some dependence on the main chain fee amount.